blazor/datagrid/print.md

```diff
@@ -11,159 +11,254 @@ documentation: ug
 
 ```cshtml
 @using Syncfusion.Blazor.Grids
+@using Syncfusion.Blazor.DropDowns
 
-<SfGrid DataSource="@Orders" Toolbar="@(new List<object>() { "Print" })" PrintMode=PrintMode.CurrentPage AllowPaging="true">
-    <GridPageSettings PageSize="8"></GridPageSettings>
+<label>Select Print Mode: </label>
+<SfDropDownList TValue="PrintMode" TItem="DropDownOrder" @bind-Value="@PrintMode" DataSource="@DropDownValue" Width="130px">
+    <DropDownListFieldSettings Text="Text" Value="Value"></DropDownListFieldSettings>
+    <DropDownListEvents ValueChange="OnValueChange" TValue="PrintMode" TItem="DropDownOrder"></DropDownListEvents>
+</SfDropDownList>
+
+<SfGrid @ref="Grid" DataSource="@Orders" Toolbar="@(new List<object>() { "Print" })" PrintMode="@PrintMode" AllowPaging="true">
+    <GridPageSettings PageSize="6"></GridPageSettings>
     <GridColumns>
         <GridColumn Field=@nameof(Order.OrderID) HeaderText="Order ID" TextAlign="TextAlign.Right" Width="120"></GridColumn>
         <GridColumn Field=@nameof(Order.CustomerID) HeaderText="Customer Name" Width="120"></GridColumn>
-        <GridColumn Field=@nameof(Order.OrderDate) HeaderText=" Order Date" Format="d" TextAlign="TextAlign.Right" Width="130" Type="ColumnType.Date"></GridColumn>
-        <GridColumn Field=@nameof(Order.Freight) HeaderText="Freight" Format="C2" TextAlign="TextAlign.Right" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(Order.ShipCity) HeaderText="Ship City" TextAlign="TextAlign.Right" Width="130" Type="ColumnType.Date"></GridColumn>
+        <GridColumn Field=@nameof(Order.ShipName) HeaderText="Ship Name" TextAlign="TextAlign.Right" Width="120"></GridColumn>
     </GridColumns>
 </SfGrid>
 
-@code{
-
+@code {
+    private SfGrid<Order> Grid;
     public List<Order> Orders { get; set; }
+    public PrintMode PrintMode { get; set; } = PrintMode.AllPages;
+
+    public class DropDownOrder
+    {
+        public string Text { get; set; }
+        public PrintMode Value { get; set; }
+    }
+    List<DropDownOrder> DropDownValue = new List<DropDownOrder>
+    {
+        new DropDownOrder() { Text = "All Pages", Value = PrintMode.AllPages },
+        new DropDownOrder() { Text = "Current Page", Value = PrintMode.CurrentPage },
+
+    };
+    public void OnValueChange(ChangeEventArgs<PrintMode, DropDownOrder> Args)
+    {
+        PrintMode = Args.Value;
+        Grid.Refresh();
+    }
 
     protected override void OnInitialized()
     {
-        Orders = Enumerable.Range(1, 75).Select(x => new Order()
-        {
-            OrderID = 1000 + x,
-            CustomerID = (new string[] { "ALFKI", "ANANTR", "ANTON", "BLONP", "BOLID" })[new Random().Next(5)],
-            Freight = 2.1 * x,
-            OrderDate = DateTime.Now.AddDays(-x),
-        }).ToList();
+        Orders = Enumerable.Range(1, 15).Select(x => new Order()
+            {
```

Comment:
move the datasource to separate cs file.

Reply:
need to use random data because we need the more records.

Reply:
records count can be increased by looping the datasource.

Reply:
move to separate cs file. use tab and separate datasource.